packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2264,9 +2265,20 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
     /**
      * When running GC in test mode, this is called to delete objects whose routes are unused. This enables testing
      * scenarios with accessing deleted content.
-     * @param unusedRoutes - The routes that are unused in all data stores in this Container.
+     * @param unusedRoutes - The routes that are unused in all data stores and blobs in this Container.
      */
     public deleteUnusedRoutes(unusedRoutes: string[]) {
+        /**
+         * When running GC in tombstone mode, this is called to tombstone datastore routes that are unused. This
+         * enables testing scenarios without actually deleting content. The content acts as if it's deleted to the
+         * external user, but the internal runtime does not delete it in summarizes, etc.
+         */
+        const tombstone = this.mc.config.getBoolean(testTombstoneKey) ?? false;
+        if (tombstone) {
+            this.dataStores.deleteUnusedRoutes(unusedRoutes, tombstone);
```

Comment:
Don't we need the same logic as below to filter based on `isBlobPath`?

Reply:
Oh I see, only datastores are passed.  Let's assert that?  Or at least put a comment... 🤔

Reply:
I think this.datastores.deleteUnusedRoutes(unusedRoutes) would fail if we passed in blob datastore paths. I haven't actually tried, I'll put a comment.

Reply:
I should add attachment blobs to my test (yikes these tests need to get complicated)

Reply:
Will do this in a later PR to add attachment blobs